content/en/users/dev-env/notebooks/data/_index.md

```diff
@@ -76,11 +76,39 @@ automatically. You can also specify a different oneprovider host if needed.
 
 ### EUDAT B2DROP
 
-[EUDAT B2DROP](https://b2drop.eudat.eu/) offers a WebDAV interface that can be
-used to mount your files from the notebooks. Files are accessed as any regular
-file from the notebooks interface or from your code. This feature requires users
-to create a client in B2DROP and provide the client's credentials to the EGI
-notebooks service.
+[EUDAT B2DROP](https://eudat.eu/services/b2drop) is a low-barrier,
+user-friendly and trustworthy storage environment which allows users
+to synchronise their active data across different desktops and to easily
+share this data with peers. EUDAT offers a free public basic instance
+of B2DROP for any researcher with a 20 GB quota.
```

Comment:
```suggestion
[EUDAT B2DROP](https://eudat.eu/services/b2drop) is a low-barrier,
user-friendly and trustworthy storage environment which allows users
to synchronise their active data across different desktops, and to easily
share this data with peers. EUDAT offers free access, for any researcher,
to a public basic instance of B2DROP, with a 20 GB quota.
```

Reply:
Not sure about the need/usefulness of the adjective `basic`.

Reply:
Is it really synchronised, meaning there is a local copy of the file, or is more like some exposure/virtual file system?

Reply:
@enolfc has the details.

From the [previous version of the docs](https://docs.egi.eu/users/dev-env/notebooks/data/#eudat-b2drop), it looks like a `WebDAV` mount.

Reply:
the `synchronised` comes from B2drop, I would just keep it like that

Reply:
Thanks!